code/modules/jobs/job_types/assistant.dm

```diff
@@ -195,3 +195,11 @@ Assistant
 	suit = /obj/item/clothing/suit/armor/roumain/shadow
 
 	head = /obj/item/clothing/head/cowboy/sec/roumain/shadow
+
+/datum/outfit/job/assistant/syndicate/cyberagent
+	name = "Junior Agent (Cybersun)"
```

Comment:
Please leave (parentheses) out of your job names.
This could be changed to "Junior Cybersun Agent" or something similar.

Reply:
Imagine being me and being stupid.